esp/src/src-react/components/Title.tsx

```diff
@@ -183,8 +178,8 @@ export const DevTitle: React.FunctionComponent<DevTitleProps> = ({
                     <Stack.Item align="center">
                         <Link href="#/activities">
                             <Text variant="large" nowrap block >
-                                <b style={{ color: (titlebarColor !== toolbarThemeDefaults.color) ? Utility.textColor(titlebarColor) : theme.palette.themeDarker }}>
-                                    ECL Watch - 9{environmentTitle !== "" && showEnvironmentTitle ? ` | ${environmentTitle}` : ""}
+                                <b title="ECL Watch - 9" style={{ color: showEnvironmentTitle && titlebarColor ? Utility.textColor(titlebarColor) : theme.palette.themeDarker }}>
```

Comment:
A few here also, the "- 9".

Reply:
This will show up as a conflict at some point.